packages/server/src/utils.ts

```diff
@@ -172,7 +198,7 @@ export function isFetchEvent(event: any): event is FetchEvent {
   return event != null && event.request != null && event.respondWith != null;
 }
 
-function configureSocket(rawRequest: NodeRequest) {
+function configureSocket(rawRequest: NodeRequest | IncomingMessage | Http2ServerRequest) {
```

Comment:
`NodeRequest` should already cover the other two. I intentionally created `NodeRequest` interface not to use `IncomingMessage`.